### PR TITLE
Update scheduling request

### PR DESCRIPTION
**Pull request by ProNinoGit120**
Updated JSON data on scheduling request

### CODE REVIEW

File: src/Components/CalendarComponents/CalendarEventCreatePopover.vue [hunk 1]
```diff
@@ -296,13 +295,19 @@ export default {
 
     getEvent(type, extra = {}) {
       const evDetails = this.details;
+      const today = new Date();
```
Comment (henryrossiter) [inline src/Components/CalendarComponents/CalendarEventCreatePopover.vue]: What is this for?

**Comment by ProNinoGit120** [comment]
To prevent that kind of error, Ben removed the old data on database.
Let fix the other issues.

**Comment by devbtech** [comment]
Request scheduling should be updated with an `event_id` for each event after a successful creation on the server.
this parameter is required for updating and deleting in `setSchedule()` event
the example of request:
`{ "schedule": { "events": [ "id": "23h5vhj32v532", // this is event id we send to frontend for each successfully created event data: { users: [] // contains profile objects with _id for selected users like it was before ... } ] } }`

**Comment by ProNinoGit120** [comment]
@devbtech Please review again and merge.

**Pull request merged**